CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased][unreleased]
 
+### Changed
+- Chore: Use native watch mode and CSS modules of esbuild #3571
```

Comment:
maybe we should also mention the dependency updates?
I don't know as most of them are just dev tooling, but maybe we did it in the past atleast for the big ones as esbuild.